frontend/packages/noobaa-storage-plugin/src/components/create-backingstore-page/create-bs.tsx

```diff
@@ -88,6 +89,19 @@ const awsRegions = [
 
 const awsRegionItems = _.zipObject(awsRegions, awsRegions);
 
+const nameLabelTooltip = (
```

Comment:
We have a ton of forms that accept a name, but don't have a tooltip like this anywhere else. I think we should apply this generally everywhere or leave it out. The problem is that different resources can have different rules.

Reply:
@shirimordechay WDYT? Would it be okay for us to leave the tooltip out?

Reply:
The question is if those other places also have a rule list that might cause error messages...
What is worst? to add those tooltip details ahead and let the user know what is a valid name (and prevent errors) or add a new type of tooltip? If the user won't have those details anywhere he can make one mistake after another and continuously receive error messages. This can be very frustrating. Showing the requirements/ rules ahead is a great way to prevent errors and frustrations. 
Is there any other implementation that handles this kind of situation?

Reply:
IMO the error messages will guide the user to come up with the correct value for a field.

Reply:
Yes, eventually the user should make it but what if the user types a name that has 3-4 different errors? will we show all those errors at once or only the first invalid rule? 

In any way, best is to prevent errors since the console error messages are not very informative at the moment and may not guide the user so well.

Reply:
@spadgett your thoughts? I believe we should let the user try without any aid( assuming that an administrator using these features s/he would be able to decode the error).

Reply:
@shirimordechay whatever we decide, we shouldn't do it for just one form since this is an issue for every form. that's my point. we should be consistent.

if we make a change, it should be a separate PR that updates all forms that have a name input

it would be even better if we can use the open api description so that the help text is *always* accurate. we've gotten out of sync in the past and shown a client hint that was not longer current with the api. that is arguably worse.

i believe we have some design stories to look at how to show field-level help in forms. we are already showing the open api doc in details pages. we should try to be consistent in how we show it for forms as well. see #2926

https://github.com/openshift/console/blob/master/frontend/public/components/utils/details-item.tsx#L51-L65

i'd rather solve this generally for all forms as a separate story

Reply:
Yes, I agree there should be a general solution for this type of issue. I would try to raise this issue at the UXD level so that hopefully this could be a reuse component in the future. 
@spadgett Where do we have other multi requirements naming in the console?  

@bipuladh Can we at least make sure that the error messages are clear enough? or this is related to the other issue we talked about?

Reply:
@shirimordechay Yeah you're right it's related to the other issue(it's send by the operator).